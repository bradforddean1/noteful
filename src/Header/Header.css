.Header {
  display: flex;
  align-items: center;
  justify-content: center;
  padding: 40px;
}

.Header a {
  font-family: Arial, Helvetica, sans-serif;
  font-size: 24px;
  text-decoration: none;
  color: black;
}
