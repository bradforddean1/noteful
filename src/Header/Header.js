import React from "react";
import "./Header.css";
import { Link } from "react-router-dom";
import PropTypes from "prop-types";

/**
 *Header w/ Link to "/"
 *@component
 */
function Header() {
  return (
    <header className="Header">
      <Link to="/">Noteful</Link>
    </header>
  );
}

export default Header;
