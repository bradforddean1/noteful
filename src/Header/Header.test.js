import React from "react";
import ReactDOM from "react-dom";
import Header from "./Header";
import { shallow } from "enzyme";
import toJson from "enzyme-to-json";
import { BrowserRouter } from "react-router-dom";

describe("Header Component", () => {
  // Smoke test
  test("Renders", () => {
    const div = document.createElement("div");
    ReactDOM.render(
      <BrowserRouter>
        <Header />
      </BrowserRouter>,
      div
    );
    ReactDOM.unmountComponentAtNode(div);
  });

  // Snapshot test
  test("Renders Default State", () => {
    const wrapper = shallow(<Header />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
