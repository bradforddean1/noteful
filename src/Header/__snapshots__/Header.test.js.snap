// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header Component Renders Default State 1`] = `
<header
  className="Header"
>
  <Link
    to="/"
  >
    Noteful
  </Link>
</header>
`;
